### DISCUSSION TITLE
Configure Blackhole to pass audio from Zoom meeting to Recordia

### DISCUSSION THREAD

**Discussion by AlecMBurton** [ExistentialAudio/BlackHole#805] [Q&A]
Hello everyone,

I'm trying to get Blackhole configured to route audio from headphone/microphone and zoom application to recordia for audio recording.

Here are my current configurations:
<img width="803" alt="Screenshot 2024-09-23 at 3 01 38 PM" src="https://github.com/user-attachments/assets/123f872b-157f-470c-868d-16c85624ef12">
<img width="800" alt="Screenshot 2024-09-23 at 3 01 49 PM" src="https://github.com/user-attachments/assets/c566b14b-78c5-4379-a84e-8d68ac7ce7ef">
<img width="797" alt="Screenshot 2024-09-23 at 3 02 48 PM" src="https://github.com/user-attachments/assets/fe7024c1-fa65-476f-95de-124ba89bc59d">

When I select Aggregate Device as the input for Blackhole, I will hear my microphone audio. When I select Blackhole 2ch as the input device I hear the zoom audio, but can't hear my own. 
<img width="273" alt="Screenshot 2024-09-23 at 3 03 47 PM" src="https://github.com/user-attachments/assets/e6f57be7-d97f-4902-8446-63905487d4c3">

Any suggestions or help would be much appreciated! Thanks a ton!

**Comment by devinroth** [comment]
I know nothing about recordia but can it record more than two channels? If so, use an aggregate with BlackHole and Microphone and record all the channels. An aggregate does not mix the audio together so they will be recorded on separate tracks.

Devin

**Reply by AlecMBurton** [reply]
Thank you so much for your reply @devinroth. From what I have seen I'm only able to make a single selection. I'm able to force mono, but I don't think that achieves what I want. I want to be able to record audio from both zoom call and the microphone of the host. Is it possible to create a single channel through blackhole? I'm also not familiar with the audio terminology so I apologize for any incorrect usage.

**Comment by devinroth** [comment]
BlackHole does not do any mixing. Either use a DAW, a different recording software, or you can try our other paid app Dipper. https://dipper.audio

Devin
